guides/source/threading_and_code_execution.md

```diff
@@ -72,7 +72,7 @@ end
 ```
 
 TIP: If you repeatedly invoke application code from a long-running process, you
-may want to wrap using the Reloader instead.
+may want to wrap using the Reloader (see below) instead.
```

Comment:
What if we link to the section instead?

```suggestion
may want to wrap using the [Reloader](#reloader) instead.
```

Reply:
Ah, I just forgot the link feature. Thanks for your suggestion! I committed them.